Python/wrfrun/unCMIP6.py

```diff
@@ -448,15 +510,15 @@ def interp_data(self):
                     Temp = [[round(ii) for ii in nested] for nested in Temp]
                     Temp = np.array(Temp)                                       
                     ds.values = Temp
-                # NOTE: This is because in MPI-ESM1-2-HR data sftlf is in percentage (either 0 or 100, not 
-                #   in between) and WRF expects 0 or 1 values. Also we do not need to worry about missing values   
-                #   as I checked and there are none of those in the sftlf data.
+                # NOTE: This is because in some model data sftlf is in percentage and WRF expects 0-1 values. 
```

Comment:
Why do we have a nested for loop in line 448/510???  how about `ds = np.round(ds / 100)`? or `ds = (ds / 100).round()` if you don't want to invoke numpy?

Reply:
Used da = (da/100).round().